website/docs/r/securityhub_standards_control.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Security Hub"
+layout: "aws"
+page_title: "AWS: aws_securityhub_standards_control"
+description: |-
+  Enable/disable Security Hub standards controls.
+---
+
+# Resource: aws_securityhub_standards_control
+
+Disable/enable Security Hub standards control in the current region.
+
+The `aws_securityhub_standards_control` behaves differently from normal resources, in that
+Terraform does not _create_ this resource, but instead "adopts" it
+into management. When you _delete_ this resource configuration, Terraform "abandons" resource as is and just removes it from the state.
+
+## Example Usage
+
+```hcl
+resource aws_securityhub_account example {}
+
+resource aws_securityhub_standards_subscription cis_aws_foundations_benchmark {
+  standards_arn = "arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0"
+  depends_on    = [aws_securityhub_account.example]
+}
+
+resource aws_securityhub_standards_control ensure_iam_password_policy_prevents_password_reuse {
+  arn             = "arn:aws:securityhub:us-east-1:078884324539:control/cis-aws-foundations-benchmark/v/1.2.0/1.10"
```

Comment:
Can use the `format("", replace())` syntax I suggested for the acceptance tests instead of hard-coded AWS account ID.